src/Accessors.jl

```diff
@@ -17,14 +17,14 @@ include("functionlenses.jl")
 include("testing.jl")
 
 if !isdefined(Base, :get_extension)
-    include("../ext/AccessorsDatesExt.jl")
-    include("../ext/AccessorsLinearAlgebraExt.jl")
-    include("../ext/AccessorsTestExt.jl")
+    include("../ext/DatesExt.jl")
+    include("../ext/LinearAlgebraExt.jl")
+    include("../ext/TestExt.jl")
 end
```

Comment:
Not this PR. But what do you think about bumping julia version to the current LTS 1.10 and getting rid of Requires and other legacy compat?